lib/dams/dams_helper.rb

```diff
@@ -838,8 +838,13 @@ def loadRdfObjects (object, className)
           begin
             o = className.find(o_pid)
             objects << o
-          rescue
+          rescue Exception => ex
             logger.warn "Error loading  #{className.to_s}: #{o_pid}"
+            # Try to reload the record when loading failed
+            if reload
+                puts "Error: #{ex}. \n Reloading #{o_pid} in class #{className.to_s}."
```

Comment:
:+1: to this approach! A couple of thoughts.

In this rescue block, in the situation where reload is true (i.e. trying a second time), then there will be two log entries which might be a bit confusing.

What if we changed the parameter from reload to something like:

`def loadRdfObjects(object, classname, tries: 2)`

then the rescue block could take advantage of the ruby `retry` keyword (see http://docs.ruby-lang.org/en/2.2.0/syntax/exceptions_rdoc.html for more info) and do something like:

```
rescue => e
  tries -= 1
  logger.warn "Error: #{e}. loading #{o_pid} in class #{className.to_s}. #{tries} tries left."
  retry if tries > 0
end
```

also this scenario should use `logger.warn`, or possibly even `logger.error`, instead of `puts ...` as this would be a problem that we would want to elevate as it would mean an existing service is down or not responding.